From dfe873ec88c3352d5c1cd0ada3c41bdf12f3c0d3 Mon Sep 17 00:00:00 2001
From: "wei.nie" <wei.nie@amlogic.com>
Date: Sat, 30 Aug 2014 17:20:39 +0800
Subject: [PATCH 5039/5965] PD#95560: fix screen crash issue when switch to
 animation,and fix uboot env can't get issue.

Change-Id: Iece2b2384ba1d04540cb529f2e84b6039bf27189

Conflicts:

	drivers/amlogic/display/osd/osd_hw.c
---
 arch/arm/boot/dts/amlogic/meson6_g18.dtd      | 6 +++---
 arch/arm/boot/dts/amlogic/meson6_g18_512M.dtd | 6 +++---
 arch/arm/configs/meson6_defconfig             | 2 +-
 3 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/arch/arm/boot/dts/amlogic/meson6_g18.dtd b/arch/arm/boot/dts/amlogic/meson6_g18.dtd
index 0bdb7d01ab36..793402381d60 100755
--- a/arch/arm/boot/dts/amlogic/meson6_g18.dtd
+++ b/arch/arm/boot/dts/amlogic/meson6_g18.dtd
@@ -743,7 +743,7 @@ void root_func(){
 		bootloader:bootloader{
 			name = "bootloader";
 			offset = <0>;
-			size = <0x60000>;
+			size = <0x100000>;
 		};
 
 //$$ MATCH "Spi_nr-part-1_match" = <&ubootenv>
@@ -752,8 +752,8 @@ void root_func(){
 //$$ L2 PROP_U32 = "size"
 		ubootenv:ubootenv{
 			name = "ubootenv";
-      offset = <0x80000>;
-      size = <0x8000>;
+      offset = <0x100000>;
+      size = <0x10000>;
     };
 	};
 
diff --git a/arch/arm/boot/dts/amlogic/meson6_g18_512M.dtd b/arch/arm/boot/dts/amlogic/meson6_g18_512M.dtd
index 4269a0da485f..9e58eeefbff5 100755
--- a/arch/arm/boot/dts/amlogic/meson6_g18_512M.dtd
+++ b/arch/arm/boot/dts/amlogic/meson6_g18_512M.dtd
@@ -742,7 +742,7 @@ void root_func(){
 		bootloader:bootloader{
 			name = "bootloader";
 			offset = <0>;
-			size = <0x60000>;
+			size = <0x100000>;
 		};
 
 //$$ MATCH "Spi_nr-part-1_match" = <&ubootenv>
@@ -751,8 +751,8 @@ void root_func(){
 //$$ L2 PROP_U32 = "size"
 		ubootenv:ubootenv{
 			name = "ubootenv";
-      offset = <0x80000>;
-      size = <0x8000>;
+      offset = <0x100000>;
+      size = <0x10000>;
     };
 	};
 
diff --git a/arch/arm/configs/meson6_defconfig b/arch/arm/configs/meson6_defconfig
index 0e0c55c645bc..0a8229612cc0 100755
--- a/arch/arm/configs/meson6_defconfig
+++ b/arch/arm/configs/meson6_defconfig
@@ -229,7 +229,7 @@ CONFIG_VSYNC_RDMA=y
 CONFIG_AM_VIDEO=y
 CONFIG_AM_TV_OUTPUT2=y
 CONFIG_FB_AM=y
-CONFIG_FB_OSD2_CURSOR=y
+CONFIG_FB_OSD2_CURSOR=n
 CONFIG_FB_OSD2_DEFAULT_WIDTH=32
 CONFIG_FB_OSD2_DEFAULT_HEIGHT=32
 CONFIG_FB_OSD2_DEFAULT_WIDTH_VIRTUAL=32
-- 
2.19.0

